easybuild/easyconfigs/u/uftp/uftp-5.0-destdir-without-usr.patch

```diff
@@ -0,0 +1,22 @@
+# removes hardcoded /usr prefix in install stage of makefile
```

Comment:
Add name of author for the patch please.

Reply:
No problem, I will